splash/browser_tab.py

```diff
@@ -889,19 +889,28 @@ def send_keys(self, text):
             self._send_word(word)
 
     EDMACRO_SHORTCUTS = {
-        'RET': ('', Qt.Key_Return),
-        'SPC': (' ', Qt.Key_Space),
-        'TAB': ('\t', Qt.Key_Tab),
-        'DEL': ('', Qt.Key_Delete),
-        'ESC': ('', Qt.Key_Escape),
-        # 'LFD': ('', 0), ## QChar.LineFeed?
-        # 'NUL': ('', 0), ## QChar.Null?
+        'RET': Qt.Key_Return,
+        'SPC': Qt.Key_Space,
+        'TAB': Qt.Key_Tab,
+        'DEL': Qt.Key_Delete,
+        'ESC': Qt.Key_Escape,
+        # 'LFD': 0, ## QChar.LineFeed?
+        # 'NUL': 0, ## QChar.Null?
+    }
+
+    QT_KEY_INPUTS = {
+        # TODO: More needed?
+        Qt.Key_Return: '\n',
+        Qt.Key_Enter: '\n',
+        Qt.Key_Space: ' ',
+        Qt.Key_Tab: '\t',
```

Comment:
It turns out that for the Return event to work it needs to input `\n`, so I added this mapping here. Actually, if we wanted to limit the number of QtKeys supported, we could do so here.